### DISCUSSION TITLE
Hunting - Table of First/Last Seen IPs/Domains

### DISCUSSION THREAD

**Discussion by ag100** [Security-Onion-Solutions/securityonion#2457] [Ideas] [answered]
Apologies if this isn't the right place, but I'm curious if anyone has built a table of first/last seen IPs/Domains in SecurityOnion?  It's something I've done w/Splunk in the past, and can be very useful as raw events age out, etc...  If no one has, I can start trying to figure out how difficult it would be, but wanted to check before duplicating efforts.  Thank you!

**Comment by dougburks** [comment, marked as answer]
You might start with an ElastAlert new_term rule as described in Issue #846.
